src/public/secure/admin.ejs

```diff
@@ -1355,7 +1359,7 @@
                                 alert(e.message);
                                 return;
                             }
-                            showSubmissionInfoModel(entry.id, entry.WADName, entry.level, entry.authorName, selectedRow[5], selectedRow[6], entry.author, entry.distributable, entry.info, entry.email, selectedRow[10]);
+                            showSubmissionInfoModel(entry.id, entry.WADName, entry.level, entry.authorName, selectedRow[5], selectedRow[6], entry.author, entry.distributable, entry.info, entry.email, selectedRow[10], entry.recordedFormat);
```

Comment:
So I guess this works as is but I will say, selecting out column indexes like this lends itself  to fragile code.  You have to remember to update the parameters here every time you change the UI.  Plus I'm still trying to convince myself that some combination of isPReviousResults and isResults won't break the column order and therefore this call here.